dnf/base.py

```diff
@@ -1620,7 +1624,7 @@ def package_install(self, pkg, strict=True):
         already_inst, available = self._query_matches_installed(q)
         if pkg in already_inst:
             _msg_installed(pkg)
-        elif not pkg in available:
+        elif not pkg in [a for b in available for a in b]:
```

Comment:
- fix pep8 warning
- use iterator instead (or even think about itertools usage in there)

something like `elif pkg not in itertools.chain.from_iterable(available)`